apps/www/components/ActionBar/FlyerTileActionBar.js

```diff
@@ -0,0 +1,117 @@
+import { useState } from 'react'
+import { IconButton, ShareIcon, ImageIcon, slug } from '@project-r/styleguide'
+import { PUBLIC_BASE_URL } from '../../lib/constants'
+import { trackEvent } from '../../lib/matomo'
+import ShareOverlay from './ShareOverlay'
+import { useTranslation } from '../../lib/withT'
+import { css } from 'glamor'
+import { postMessage } from '../../lib/withInNativeApp'
+import { getShareImageUrls } from '../Article/Flyer'
+
+const ShareButton = ({ meta, tileId, inNativeApp }) => {
+  const [overlay, showOverlay] = useState(false)
+  const { t } = useTranslation()
+
+  const urlToShare = new URL(meta.path, PUBLIC_BASE_URL)
+  urlToShare.searchParams.set('share', tileId)
+  const url = urlToShare.toString()
+
+  return (
+    <>
+      <IconButton
+        label={t('article/actionbar/share')}
+        labelShort={t('article/actionbar/share')}
+        Icon={ShareIcon}
+        onClick={(e) => {
+          e.preventDefault()
+          trackEvent(['ActionBar', 'shareJournalLink', url])
+          if (inNativeApp) {
+            postMessage({
+              type: 'share',
+              payload: {
+                title: 'Republik: Journal',
+                url,
+                subject: '',
+                dialogTitle: 'Link teilen',
+              },
+            })
+            e.target.blur()
+          } else {
+            showOverlay(!overlay)
+          }
+        }}
+      />
+      {!!overlay && (
+        <ShareOverlay
+          onClose={() => showOverlay(false)}
+          url={url}
+          title={t('article/actionbar/share')}
+          tweet=''
+          emailSubject={t('article/share/emailSubject', {
+            title: meta.title,
+          })}
+          emailBody=''
+          emailAttachUrl
+        />
+      )}
+    </>
+  )
+}
+
+const DownloadButton = ({ documentId, meta, tileId }) => {
+  const { t } = useTranslation()
+  const DEFAULT_LABEL = t('article/actionbar/download')
+  const [downloadLabel, setDownloadLabel] = useState(DEFAULT_LABEL)
+
+  const { screenshotUrl, shareImageUrl } = getShareImageUrls(
+    documentId,
+    meta,
+    tileId,
+    true,
+  )
+
+  const downloadScreenshot = async (e) => {
+    e.preventDefault()
+    setDownloadLabel(t('article/actionbar/download/processing'))
+    trackEvent(['ActionBar', 'downloadJournalBlock', screenshotUrl])
+    const image = await fetch(shareImageUrl)
+    const imageBlog = await image.blob()
+    const imageURL = URL.createObjectURL(imageBlog)
```

Comment:
Blobbing for download not so good (App won't handle it well?) and whole Spiel w/ "wird heruntergeladen..." could be kicked out.

We found [a decent approach for audio files](https://github.com/republik/plattform/blob/8229dd284b4720e9ddfebc9f5b49ec4e8c203b52/apps/www/components/Audio/helpers/DownloadAudioSource.ts#L9-L18). It would need that code snippet and assets server to add `Content-Disposition` header if `&download=1` is appended.